pkg/controller/datadogagent/clusterchecksrunner_test.go

```diff
@@ -40,6 +66,20 @@ func clusterChecksRunnerDefaultPodSpec() corev1.PodSpec {
 
 func clusterChecksRunnerDefaultVolumeMounts() []corev1.VolumeMount {
 	return []corev1.VolumeMount{
+		{
+			Name:      datadoghqv1alpha1.ConfdVolumeName,
```

Comment:
I think we don't need to mound the Confd volume, because the `ClusterCheckRunner` only get the config from the `ClusterAgent`.